gm4_lightning_in_a_bottle/data/gm4_lightning_in_a_bottle/functions/load.mcfunction

```diff
@@ -0,0 +1,8 @@
+execute if score gm4 load.status matches 1 run scoreboard players set gm4_lightning_in_a_bottle load.status 1
```

Comment:
This doesn't prevent the module from loading if one of the dependencies is missing